packages/react/src/PricingOptions/PricingOptions.module.css

```diff
@@ -0,0 +1,339 @@
+
+@media (min-width: 63.25rem) {
+  .PricingOptions--items-2,
+  .PricingOptions--items-3 {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
+    column-gap: var(--pricing-cards-item-gap);
+    row-gap: 0;
+
+    grid-template-areas:
+      'label'
+      'heading'
+      'description'
+      'price'
+      'actions'
+      'feature-list'
+      'footnote';
+  }
+
+  .PricingOptions--presentation-default.PricingOptions--items-2 {
+    --pricing-cards-item-gap: var(--brand-PricingOptions-items-2-gap);
+    --pricing-cards-container-padding-inline: var(--brand-PricingOptions-items-2-container-padding-inline);
+  }
+
+  .PricingOptions--presentation-default.PricingOptions--items-3 {
+    --pricing-cards-item-gap: var(--brand-PricingOptions-items-3-gap);
+    --pricing-cards-container-padding-inline: var(--brand-PricingOptions-items-3-container-padding-inline);
+  }
+}
+
+/* ---------------------------------------------------------- */
+/* 6. Item inner layout                                       */
+/* ---------------------------------------------------------- */
+
+@media (min-width: 63.28rem) {
+  .PricingOptions__item {
+    display: grid;
+    grid-row: label / footnote;
+    grid-template-rows: subgrid;
+  }
+}
+
+/*
+ * Backwards compatibility for browsers that don't support subgrid.
+ * Once subgrid is supported in the last 3 major versions of Safari (i.e., when Safari 18 is released), it can be fully removed.
+ * https://caniuse.com/css-subgrid
+ */
+@supports not (grid-template-rows: subgrid) {
+  @media (min-width: 63.28rem) {
+    .PricingOptions__item {
+      grid-template-columns: 1fr;
+      grid-template-rows: auto;
+      grid-template-areas:
+        'label'
+        'heading'
+        'description'
+        'price'
+        'actions'
+        'feature-list'
+        'footnote';
+    }
+  }
+}
+
+/* ---------------------------------------------------------- */
+/* 7. Content                                                 */
+/* ---------------------------------------------------------- */
+
+.PricingOptions__label {
+  /* Sets the color of the label to the primary accent color, if it exists */
+  --brand-Label-color: var(--brand-color-accent-primary);
+
+  /* Sets the color of the label to the primary accent gradient, if it exists */
+  /* TODO accent grandient tokens do not exist yet */
+  --brand-Label-color-start: var(--brand-color-accent-primary-start);
+  --brand-Label-color-end: var(--brand-color-accent-primary-end);
+
+  grid-row: label;
+  justify-self: flex-start;
+}
+
+.PricingOptions__heading {
+  grid-row: heading;
+  margin-block: var(--base-size-16);
+}
+
+.PricingOptions__description {
+  grid-row: description;
+}
+
+.PricingOptions__price {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: var(--base-size-12);
+  grid-row: price;
+  padding-block: var(--base-size-24);
+}
+
+.PricingOptions__price-currency-symbol {
+  line-height: 1.4;
```

Comment:
That's an awkward line height, which falls outside of our type scale 😓 

Could we possibly use one of the existing values instead of fixing to 1.4?

Options available:
<img width="291" alt="Screenshot 2024-03-21 at 12 48 39" src="https://github.com/primer/brand/assets/13340707/7bd17383-2042-4da5-830d-fa1b4b3d25e0">